arch/arm/src/armv7-r/arm_syscall.c

```diff
@@ -33,9 +33,10 @@
 #include <nuttx/arch.h>
 #include <nuttx/sched.h>
 
-#include "signal/signal.h"
+#include "addrenv.h"
 #include "arm.h"
 #include "arm_internal.h"
+#include "signal/signal.h"
```

Comment:
don't need include addrenv.h and signal/signal.h

Reply:
Depends by: nxsig_unmask_pendingsignal